sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -140,6 +141,62 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("check outputs of expression examples") {
+    def unindentAndTrim(s: String): String = {
+      s.replaceAll("\n\\s+", "\n").trim
+    }
+    val exampleRe = """^(.+);\n(?s)(.+)$""".r
+    val ignoreSet = Set(
+      // One of examples shows getting the current timestamp
+      "org.apache.spark.sql.catalyst.expressions.UnixTimestamp",
+      // Random output without a seed
+      "org.apache.spark.sql.catalyst.expressions.Rand",
+      "org.apache.spark.sql.catalyst.expressions.Randn",
+      "org.apache.spark.sql.catalyst.expressions.Shuffle",
+      "org.apache.spark.sql.catalyst.expressions.Uuid",
+      // The example call methods that return unstable results.
+      "org.apache.spark.sql.catalyst.expressions.CallMethodViaReflection",
+      // Fails on parsing `SELECT 2 mod 1.8`:
```

Comment:
What should I do for all the exceptions? @dongjoon-hyun @srowen Open a separate JIRA ticket per-each case?

Reply:
I think it's fine to just fix what you have so far. I think it's fine to fix additional ones here, too. I don't think you need to fix each of the individually unless you feel they're logically distinct.

Reply:
I have fixed 3 out of 4. I will open a ticket for the last one.

Reply:
https://issues.apache.org/jira/browse/SPARK-29264